### PR TITLE
Add sign and send transaction.

### PR DESCRIPTION
**Pull request by nickcruz**
Screen Shot 2022-01-12 at 6 08 12 PM

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -168,6 +173,23 @@ export default function App() {
       addLog("[error] signMessage: " + JSON.stringify(err));
     }
   };
+  const signAndSendTransaction = async () => {
+    try {
+      const transaction = await createTransferTransaction();
+      if (!transaction) return;
+      const { signature } = await provider.signAndSendTransaction(transaction);
```
Comment (fragosti) [inline src/App.tsx:180]: Just an FYI, all our sample code should probably be using `signAndSendTransaction` unless it is specifically demonstrating those other methods. That was the case at some point but I think it got removed.

Comment (nickcruz) [reply src/App.tsx:180]: This sandbox code is mostly demonstrating the other methods, should I remove the other sign methods other than `signAndSendTransaction`?

Comment (eliot404) [reply src/App.tsx:180]: Hi i am developing a mint site with phantom and sol. On devnet when I send sol to another wallet the transaction get confirmed but on the reciever end i don't get the sols. So what will be the reason? I am following this

https://codesandbox.io/s/nifty-lewin-4bvd8?file=/src/App.tsx

**Comment by bfriel** [comment]
Just saw this never got merged back in the day, sorry Nick 😭

**Pull request closed without merging**